boltdesignsystem/bolt
# Blockquote | Docs | Add Jest tests and documentation

## Jira

http://vjira2:8080/browse/BDS-970

## Summary

Update readme and add tests for bolt-blockquote

## Details

(Explain the changes in enough detail fo reviewers to understand.  Raise any questions for reviewers to consider.)

## How to test

Run this code locally and check if all [tests](https://github.com/bolt-design-system/bolt/pull/1152/files#diff-372cba9a19220ba9e14566ff8b83f1b8) are pass and check if [readme](https://github.com/bolt-design-system/bolt/pull/1152/files#diff-8c63325f3ae2d182cb893cfe4153b9b2) file.
